CMake/activate_venv.bat.in

```diff
@@ -5,7 +5,7 @@
 @ECHO OFF
 setlocal EnableExtensions EnableDelayedExpansion
 set "OLDPATH=%PATH%"
-activate @VENV_DIR@
+conda activate @VENV_DIR@
```

Comment:
Might be worth looking for the `$CONDA_EXE` environment variable and falling back on hoping that conda is in the PATH.

Reply:
That sounds like a much better direction. Will do.